ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -105,19 +136,19 @@ public void onInitializeSuccess() {
               @Override
               public void onInitializeError(String errorMessage) {
                 Log.d(TAG, "SDK init failed: " + VungleBannerAdapter.this);
-                VungleListener listener = getVungleListener();
                 mVungleManager.removeActiveBannerAd(mPlacementId);
-                if (mPendingRequestBanner && listener != null) {
-                  listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
+                if (mPendingRequestBanner && bannerListener != null && bannerAdapter != null) {
```

Comment:
bannerListener checks have no sense.
But as @HuFangshuai mentioned we can't hold strong reference to bannerListener due to leak.